src/types.ts

```diff
@@ -1,14 +1,42 @@
 import * as azdata from 'azdata';
 
+/**
+ * Contains success information, a sessionId to be used when requesting
+ * expansion of nodes, and a root node to display for this area
+ */
 export interface CreateSessionResponse {
+	/**
+	 * Unique Id to use when sending any requests for objects in the tree
+	 * under the node
+	 */
 	sessionId: string;
 }
 
+/**
+ * Information returned from a createSessionRequest. Contains success information, a sessionId to be used
```

Comment:
It's seems incorrect to return a parameters object from a function.  Typically, parameters are input to the method and a response/result object is the output.  As such, the comment seems incorrect (or the interface name) and the error fields seem out of place.

Reply:
These aren't my changes, they are from Cheena's PR.